insights/specs/default.py

```diff
@@ -707,6 +707,7 @@ class DefaultSpecs(Specs):
     x86_ibrs_enabled = simple_file("sys/kernel/debug/x86/ibrs_enabled")
     x86_pti_enabled = simple_file("sys/kernel/debug/x86/pti_enabled")
     x86_retp_enabled = simple_file("sys/kernel/debug/x86/retp_enabled")
+    xfs_quota_statue = simple_command("/sbin/xfs_quota -x -c 'state -gu'")
```

Comment:
You also need to add its `simple_file` to the `insights_archive.py` if it also can be collected via uploader.json